tests/tests_geomstats/test_sasaki_metric.py

```diff
@@ -0,0 +1,33 @@
+"""Unit tests for the Sasaki metric."""
+
+import geomstats.tests
+from tests.conftest import Parametrizer
+from tests.data.sasaki_metric_data import SasakiMetricTestData
+from tests.geometry_test_cases import TestCase
+
+
+class TestSasakiMetric(TestCase, metaclass=Parametrizer):
```

Comment:
Could you add a test specifically for the `grad` (since we were discussing the rightness of the formulas). I think the `geodesic_discrete` test is not enough to validate the gradient because if I'm not mistaken it evaluates to zero there.

Reply:
Good catch! I added some more data, s.t. the geodesic_discrete test now actually performs some steps, i.e. the gradient is not zero right away.